tool_collections/kraken/kraken/kraken.xml

```diff
@@ -47,7 +50,10 @@
         #if $split_reads:
             --classified-out "${classified_out}" --unclassified-out "${unclassified_out}"
         #end if
-        --output "${output}"
+
+        ## The --output option was changed to redirect as it does not work properly is some situations
```

Comment:
Anton can we please increase the version number of this tool. I guess this is needed if change the output ...